src/lib/stores.ts

```diff
@@ -1,3 +1,47 @@
 import { writable } from 'svelte/store';
 
-export const xsrf = writable('');
+const xsrf = writable('');
+
+const createAlerts = () => {
+  const { subscribe, set, update } = writable([]);
+
+  const FADE_DURATION = 500;
+  const CLOSE_DELAY = 5000;
+
+  const dismiss = (alertID: number) => {
+    update((alerts) => alerts.filter((alert) => alert.id != alertID));
+  };
+
+  return {
+    subscribe,
+    push: (
+      message: string,
+      type: 'ERROR' | 'WARNING' | 'SUCCESS',
+      fade = false
+    ) =>
+      update((alerts) => {
+        const newAlert = {
+          message,
+          type,
+          fade,
+          id: Math.round(Math.random() * 10000),
```

Comment:
No. Use UUID or something other that's actually predictably non-clashing.

Reply:
yolo